code/game/machinery/recharger.dm

```diff
@@ -1,70 +1,78 @@
-//This file was auto-corrected by findeclaration.exe on 25.5.2012 20:42:31
-
 /obj/machinery/recharger
 	name = "recharger"
 	icon = 'icons/obj/stationobjs.dmi'
 	icon_state = "recharger0"
+	desc = "A charging dock for energy based weaponry."
 	anchored = 1
 	use_power = 1
-	idle_power_usage = 40
-	active_power_usage = 2500
+	idle_power_usage = 4
+	active_power_usage = 250
```

Comment:
As with power cells, would this be the thing to change to have charging weapons take up power based on their internal cell size, and to have losses in doing so?